mycroft/client/wifisetup/main.py

```diff
@@ -38,6 +42,29 @@
 
 LOG = getLogger("WiFiClient")
 
+SCRIPT_DIR = dirname(realpath(__file__))
+
+
+def var_dump(var, prefix=''):
+    my_type = '[' + var.__class__.__name__ + '(' + str(len(var)) + ')]:'
+    print(prefix, my_type)
+    prefix += '    '
+    for i in var:
+        if type(i) in (list, tuple, dict, set):
+            var_dump(i, prefix)
+        else:
+            if isinstance(var, dict):
+                print(prefix, i, ': (', var[i].__class__.__name__, ')', var[i])
+            else:
+                print(prefix, '(', i.__class__.__name__, ')', i)
+
+
+def cli_no_output(*args):
```

Comment:
P.S.  I hope none of the responses here seem snippy.  I was writing them around 5am.  :)